### PR TITLE
Add google_logging_metric to version 5 upgrade doc for #8780

### PR DESCRIPTION
**Pull request by pengq-google**
Add google_logging_metric to version 5 upgrade doc for #8780

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none
Add google_logging_metric to version 5 upgrade doc for #8780
```

### CODE REVIEW

**Comment by pengq-google** [comment]
@c2thorn Thanks! :)

**Comment by pengq-google** [comment]
Sorry I am not quite sure why it is still some conflicts. Is it ok that I can revisit it tomorrow? I have another office hour with your Terraform team. Thank you and your team so much!

**Comment by c2thorn** [comment]
> Sorry I am not quite sure why it is still some conflicts. Is it ok that I can revisit it tomorrow? I have another office hour with your Terraform team. Thank you and your team so much!

There may be multiple commits simultaneously that are being merged, it's a busy time for that file. Copying your text, pulling from the `main` branch, then adding your text should ensure no conflicts (for the time being). You can also go to the file here: https://github.com/GoogleCloudPlatform/magic-modules/blob/main/mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown and just hit the edit button from the browser

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 1]
```diff
@@ -385,6 +385,17 @@ If you were relying on accessing an individual flag by index (for example, `goog
...
+### Additional `bucket_options` subfields are now properly required
+
+When setting the `bucket_options` block, the following fields may be required:
+
+* `num_finite_buckets`, `width`, and `offset` are now required when `bucket_options.linear_buckets` is set.
+
+* `num_finite_buckets`, `growth_factor`, and `scale` are now required when `bucket_options.exponential_buckets` is set.
+
+Previously these fields should have been required but were not, which allowed for invalid `google_logging_metric` configurations.
 ## Resource: `google_logging_project_sink`
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:399]:
```suggestion
Previously these fields should have been required but were not, which allowed for invalid `google_logging_metric` configurations.

## Resource: `google_logging_project_sink`
```

File: mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown [hunk 2]
```diff
@@ -423,6 +434,7 @@ resource "google_project_iam_binding" "gcs-bucket-writer" {
   ]
 }
 ```
+
```
Comment (c2thorn) [inline mmv1/third_party/terraform/website/docs/guides/version_5_upgrade.html.markdown:437]:
```suggestion
```

**Review by c2thorn — APPROVED** [review]

**Pull request merged**